deepfence_agent/Dockerfile

```diff
@@ -75,14 +75,15 @@ RUN apt-get update --allow-insecure-repositories \
     && cd /tmp \
     && chmod +x /usr/local/bin/start_services \
     && apt-get clean && apt-get -y autoremove && rm -rf /var/lib/apt/lists/* \
-    && curl -fsSLOk https://github.com/deepfence/vessel/releases/download/v0.9.0/vessel_v0.9.0_linux_amd64.tar.gz \
-    && tar -xzf vessel_v0.9.0_linux_amd64.tar.gz \
+    && curl -fsSLOk https://github.com/deepfence/vessel/releases/download/v0.11.0/vessel_v0.11.0_linux_amd64.tar.gz \
```

Comment:
Can we replace this with a command to download the current latest version of vessel ? If not, whenever we release an update to vessel, we will have to change this line in code and update agent too. @noboruma one more for the plugin I suppose ?

Reply:
I would not want latest vessel version to be downloaded automatically.
I need to use v0.10.0 in TM v1.5 and v0.11.0 in TM v2.
(Latest vessel may not work with TM v1.5 because golang version is different)

Reply:
go.mod is a potential solution path

Reply:
We need to agree on our update strategy - I agree with Ramanan, we don't want any automatic upgrade as this might affect users, we need to ask them first and then proceed.
With that being said, we definitely need a way to upgrade vessel. Here we have two ways:
- Either we do a generic upgrade: all binaries are bundled together and we extract them once together
- Or we do partial upgrades for each individual components/plugins: binaries are upgrade separately

Both have advantages - problem with second approach is we need to make sure every piece of up-gradable code is a plugin, which is not the case here with vessel (it's a utils tool)